src/modules/75-cf/utils/__tests__/UsageLimitUtils.test.ts

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { BannerType } from '@common/layouts/Constants'
+import { getBannerText } from '../UsageLimitUtils'
+
+describe('Usage Limit Utils', () => {
+  describe('when optional data are undefined', () => {
+    test('it should not display message or banner when count is undefined', () => {
+      const count = undefined
+      const limit = 25000
+      const expectedMessage = undefined
+      const getString = jest.fn().mockReturnValue(expectedMessage)
+
+      const additionalLicenseProps = {
+        isFreeEdition: true,
+        isEnterpriseEdition: false,
+        isTeamEdition: false
+      }
+      const { message, bannerType } = getBannerText(getString, additionalLicenseProps, count, limit)
+
+      expect(message).toEqual(expectedMessage)
+      expect(bannerType).toEqual(BannerType.INFO)
+    })
+
+    test('it should not display message or banner when limit is undefined', () => {
+      const limit = undefined
+      const count = 24000
+      const expectedMessage = undefined
+      const getString = jest.fn().mockReturnValue(expectedMessage)
+
+      const additionalLicenseProps = {
+        isFreeEdition: true,
+        isEnterpriseEdition: false,
+        isTeamEdition: false
+      }
+      const { message, bannerType } = getBannerText(getString, additionalLicenseProps, count, limit)
+
+      expect(message).toEqual(expectedMessage)
+      expect(bannerType).toEqual(BannerType.INFO)
+    })
+
+    test('it should not display message or banner when limit and count is undefined', () => {
+      const limit = undefined
+      const count = undefined
+      const expectedMessage = undefined
+      const getString = jest.fn().mockReturnValue(expectedMessage)
+
+      const additionalLicenseProps = {
+        isFreeEdition: true,
+        isEnterpriseEdition: false,
+        isTeamEdition: false
+      }
+      const { message, bannerType } = getBannerText(getString, additionalLicenseProps, count, limit)
```

Comment:
Instead of passing `undefined` for these two, you can just not pass them
```suggestion
      const { message, bannerType } = getBannerText(getString, additionalLicenseProps)
```

Reply:
doneee